docs/install_guides/installing-openvino-linux.md

```diff
@@ -153,184 +130,118 @@ These dependencies are required for:
 
 ## <a name="set-the-environment-variables"></a>Step 3: Configure the Environment
 
-You must update several environment variables before you can compile and run OpenVINO™ applications. Set persistent environment variables as follows, using vi (as below) or your preferred editor:
+You must update several environment variables before you can compile and run OpenVINO™ applications. Set environment variables as follows:
 
-1. Open the `.bashrc` file in `/home/<USER>`:
-   ```sh
-   vi ~/.bashrc
-   ```
+```sh
+source <INSTALL_DIR>/intel/openvino_2022/bin/setupvars.sh
+```  
 
-2. Press the **i** key to switch to insert mode.
+If you have more than one OpenVINO version on your machine, you can easily switch its version by sourcing `setupvars.sh` of your choice.
 
-3. Add this line to the end of the file:
-   ```sh
-   source /opt/intel/openvino_2022/bin/setupvars.sh
-   ```
+> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/intel/openvino_2022/bin/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO versions on your machine and want to switch among them, as each may require different setup.
 
-4. Save and close the file: press the **Esc** key and type `:wq`.
+The environment variables are set. Next, you can download some additional tools.
 
-5. To verify the change, open a new terminal. You will see `[setupvars.sh] OpenVINO environment initialized`.
+## <a name="model-optimizer">Step 4 (Optional): Download additional components
```

Comment:
Can we add here information on how to download OpenCV using a script supplied by installer? 
+ @mshabunin to provide instructions.